Dockerfile

```diff
@@ -0,0 +1,64 @@
+# syntax = docker/dockerfile:1
+
+# Make sure RUBY_VERSION matches the Ruby version in .tool-versions
+FROM registry.docker.com/library/ruby:3.2.6-slim AS base
+
+# Rails app lives here
+WORKDIR /rails
+
+ARG RAILS_MASTER_KEY
+# Set production environment
+ENV RAILS_ENV="production" \
+    RAILS_MASTER_KEY=$RAILS_MASTER_KEY \
```

Comment:
This persists the Master Key from Buildtime in the final image and therefore would leak this secret. This Env MUST be set only in the `build` image